packages/nx-plugin/src/executors/playwright-test/executor.ts

```diff
@@ -0,0 +1,24 @@
+import { execSync } from 'child_process';
+import { ExecutorContext } from '@nx/devkit';
+import { PlaywrightTestExecutorSchema } from './schema';
+
+export default async function runExecutor(
```

Comment:
can we tweak the api to follow something like this ? ( we can omit invoking nx core executor in order to keep this simple for now )

<img width="1229" alt="image" src="https://github.com/microsoft/fluentui-contrib/assets/1223799/b3e096ab-fef0-4f7e-8a1c-1c4507ba6a3f">